spec.md

```diff
@@ -100,6 +110,18 @@ interface ExpressionStatement <: Statement {
 
 An expression statement, i.e., a statement consisting of a single expression.
 
+## Directive
+
+```js
+interface Directive <: ExpressionStatement {
+    directive: string;
+}
+```
+
+A directive from the directive prologue of a script or function.
+The `directive` property is the raw string source of the directive.
+This source includes single or double wrapping quotes and may not contain an EscapeSequence or LineContinuation..
```

Comment:
I see your point about `use\x20strict` being valid directive, but your other examples can be totally covered by boolean `directive` and I don't really see arguments why you claim it can't be. It's just a task of parser to mark valid directives with corresponding flag, and it's very easy to add.

As for escapes - having yet another CST property (`directive` string) in tree, when we have `value` and semi-standardized `raw` already. It would violate principles of this spec, and make life much harder for those who write AST transformations as they will need to synchronize all the places that represent essentially the same value in different forms (both encoded and unencoded).

This should be handled as part of #41 in generic way and it's either standardizing already-used `raw` in `Literal`s or treating CST things (like this one) as API or some other way which, again, would not violate our principles.